.github/workflows/scripts/unix_test.sh

```diff
@@ -20,7 +20,9 @@ fi
 python3 -m pip install dist/*.whl
 if [ -z "$GPU_TEST" ]; then
     python3 -m pip install -r requirements_test.txt
-    python3 -m pip install torch
+    if [[ $PY != *"3.10"* && $PY != *"py310"* ]]; then
```

Comment:
I suggest making a `requirements_torch.txt` which contains:
```
torch; python_version < '3.10'
```
Let `pip` decide whether to install based on the python version.

Reply:
Or more simple, via command line:

```
python3 -m pip install "torch; python_version < '3.10'"
```